fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
this extended status, is not just 'extended' it is specific to cymru, right?

I just wonder if it needs to be spelled out as such (e.g. s/extended/cymru/ throughout) since someone might come up with another "extended" status output based on another service